kubespawner/spawner.py

```diff
@@ -109,6 +108,26 @@ def _namespace_default(self):
         """
     ).tag(config=True)
 
+
+    node_selector = Dict(
```

Comment:
Can this be a separate commit / PR? We need to be careful to support both simple property based selectors and more complex ones...